src/Sulu/Component/Util/XmlUtil.php

```diff
@@ -45,14 +45,14 @@ public static function getValueFromXPath($path, \DOMXPath $xpath, \DOMNode $cont
      * @param string $path
      * @param \DOMNode $context
      *
-     * @return bool|null|string|mixed
+     * @return bool|null
      */
-    public static function getBooleanValueFromXPath($path, \DOMXPath $xpath, \DOMNode $context = null, $default = null)
+    public static function getBooleanValueFromXPath($path, \DOMXPath $xpath, \DOMNode $context = null, $default = null): ?bool
```

Comment:
Everywhere in the code the default value is provided and it would make sense that it is a boolean as well. I would suggest to have the `$default` parameter be typed and default to false (which would be null as a boolean under non-strict types).